active_help.go

```diff
@@ -29,6 +30,8 @@ const (
 	activeHelpGlobalDisable = "0"
 )
 
+var activeHelpEnvVarPrefixSubstRegexp = regexp.MustCompile(`[^A-Z0-9_]`)
```

Comment:
Regarding moving to `activeHelpEnvVar`, if we do that, I think we'll end up just pulling it back out of there when #1938 materializes on top of this -- at that point I believe we'd end up compiling the regex several times otherwise. I'm not sure offhand if there are code paths in which we'd end up doing that already here.

Regarding the comment, I'm not sure I understand -- it feels somewhat out of place to me to comment that those "will need to be replaced", because that's what the regex is already used to do. As a user, I'd not be sure if I'd need to do something extra on top of that, and how would I end up finding the doc in the first place :thinking:

Reply:
I forgot about of #1938, so itt makes sense to leave it as is.

And ok about not adding a comment if you feel it will be more confusing.